### PR TITLE
[M1766] make cloudtunnel for pg access

### PR DESCRIPTION
**Pull request by saanobhaai**

### CODE REVIEW

**Comment by saanobhaai** [comment]
[pgadmin access](https://trello.com/c/O0na1gQt/1766-pgadmin-access)

File: Makefile [hunk 1]
```diff
@@ -104,15 +104,31 @@ test:
...
-		--region us-east-1 \
+		--profile mermaid \
 		--cluster mermaid-api-infra-common-MermaidApiClusterB0854EC6-xitj9XbqTwap \
 		--task $(taskid) \
 		--container MermaidAPI \
 		--command "/bin/bash" \
 		--interactive
+
+cloudtunnel:
+	$(eval cluster=mermaid-api-infra-common-MermaidApiClusterB0854EC6-xitj9XbqTwap)
```
Comment (gridcell) [inline Makefile:124]: Are we providing to much infrastructure details here?

cc: @aspcanada

Comment (saanobhaai) [reply Makefile:124]: Yeah I don't like this either, but not sure what to do... Maybe move to another file, out of version control?

Comment (aspcanada) [reply Makefile:124]: 1. I think the resource ID's should be in ENV VARs.
2. Perhaps some of the ports should be masked out too?

**Review by gridcell — APPROVED** [review]

**Pull request merged**